mozilla/kitsune
# [bug 1019649] Add news section to community hub.

I moved around some CSS for reusage. Also deleted a bunch of stuff that was for the static mockup and removed the under construction banner.

To verify:
- Run tests
- Create the community news document and make sure it shows up on the /community page. You can do that with this scripty:

```
from kitsune.wiki.tests import document, revision
d = document(title='Community Hub News', slug='community-hub-news', save=True)
r = revision(document=d, content="""=splendid=
sumodev is leet
==awesome==
sumo rocks!""", is_approved=True, save=True)
d.current_revision = r
d.save()
```

r? - @rehandalal
